### PR TITLE
Add keyboard shortcuts button

### LINKED ISSUE #757: Add a button to bring up keyboard shortcut help
We should add a button to bring up the keyboard shortcut menu. We should add this to the "Support" menu dropdown. However, "support" no longer feels as applicable, so let's do the following:

* Add a "Keyboard Shortcuts" item to the support dropdown on the map page
* Use a keyboard icon like this for the "Keyboard Shortcuts" item: 
* Rename the dropdown to "Resources"

### PR DESCRIPTION
**Pull request by kevinearldenny**
## Overview

- Renames the Support menu to Resources
- Adds a button to the Resources menu to Show Keyboard Shortcuts

### Checklist

- [x] Description of PR is in an appropriate section of `CHANGELOG.md` and grouped with similar changes, if possible

### Demo

Optional. Screenshots, examples, etc.

### Notes

- ~Pending a keyboard icon from @jfrankl~
- @tgilcrest This now creates a link on the homepage header which is broken - since the Keyboard Shortcuts modal is only accessible thru the Map component. Does it make sense to hide this new button unless the user is within a project? 

## Testing Instructions

- Open a map; click the Resources dropdown and 'Show keyboard shortcuts'
- Expect keyboard shortcuts modal to be displayed

Closes #757

### CODE REVIEW

File: src/client/icons.ts [hunk 1]
```diff
@@ -104,6 +104,11 @@ const icons = {
     path:
       "M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"
   },
+  keyboard: {
+    viewBox: "0 0 512 512",
+    path:
+      "TBD"
+  },
```
Comment (kevinearldenny) [inline src/client/icons.ts:111]: We can swap in the keyboard icon SVG from @jfrankl whenever it's ready

**Comment by maurizi** [comment]
> @tgilcrest This now creates a link on the homepage header which is broken - since the Keyboard Shortcuts modal is only accessible thru the Map component. Does it make sense to hide this new button unless the user is within a project?

Yeah lets not add a broken button to the application. Only showing it on the project screen makes sense to me.

File: src/client/components/SupportMenu.tsx [hunk 2]
```diff
@@ -51,6 +59,16 @@ const SupportMenu = (props: SupportProps) => {
...
+          {project && (
+            <li key={UserMenuKeys.KeyboardShortcuts}>
+              <MenuItem value={UserMenuKeys.KeyboardShortcuts}>
+                <Styled.a target="_blank" sx={style.menuListItem} onClick={showKeyboardShortcuts}>
+                  <Icon name="keyboard" sx={style.menuListIcon} />
+                  Show keyboard shortcuts
+                </Styled.a>
+              </MenuItem>
+            </li>
+          )}
```
Comment (kevinearldenny) [inline src/client/components/SupportMenu.tsx:71]: @maurizi Currently hiding the menu option unless there is a project currently selected 👍

**Comment by jfrankl** [comment]
I just pushed a quick style commit on this. Left aligned the actions so they are closer to the commands. Formatted the commands a little differently with a fixed width typeface.

Screen Shot 2021-05-26 at 9 19 28 PM

**Review by maurizi — APPROVED** [review]
👍

**Pull request merged**